tests/integration/test_datatable_citations.py

```diff
@@ -28,14 +28,5 @@ def test_citations(app):
         response = client.get('/ajax/citations?recid=712925&endpoint=literature')
         assert response.status_code == 200
 
-        response = client.get('/ajax/citations?recid=1319638&endpoint=literature')
-        assert response.status_code == 200
-
         response = client.get('/ajax/citations?recid=452060&endpoint=literature')
         assert response.status_code == 200
```

Comment:
IMO you can also remove this guy, we only need to test one record here (and write a better test).